netdisco/const.py

```diff
@@ -27,6 +27,7 @@
 DECONZ = "deconz"
 TIVO_DVR = "tivo_dvr"
 FREEBOX = "freebox"
+XBOX_ONE = "xbox_one"
```

Comment:
We should rename the file as SmartGlass might not be xbox one specific ? I think that xbox_smartglass would be a better name.

Reply:
SmartGlass protocol actually IS Xbox One specific, but I am fine to do that change.

Reply:
It's XBox One specific until it is reused in XBox Two (or whatever) and it no longer is 😉

Reply:
fair point!